infomin.csl

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Infomin</title>
+    <title-short>Infomin</title-short>
+    <id>http://www.zotero.org/styles/infomin</id>
+    <link href="http://www.zotero.org/styles/infomin" rel="self"/>
+    <link href="http://www.infomin.co.cu/index.php/i/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Rafael Cervantes Beyra</name>
+      <email>cervantes@unah.edu.cu</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <eissn>1992-4194</eissn>
+    <updated>2019-03-02T20:22:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="anonymous">Anon.</term>
+      <term name="no date">n.d.</term>
+      <term name="retrieved">Available</term>
+      <term name="accessed">Consulted</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="never">
+        <name-part name="family"/>
+        <name-part name="given"/>
+      </name>
+      <substitute>
+        <names variable="editor">
+          <name and="symbol" name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="never"/>
+          <label form="short" prefix=" (" suffix=")."/>
```

Comment:
add the label just like this to the author (where it's empty), then use just `<names variable="editor"/>` (and translator, interviewer, container-author) in substitute.